server/server.js

```diff
@@ -1,12 +1,57 @@
 const express = require('express');
 let app = express();
-let port = 4006;
 const bodyParser = require('body-parser');
+const db = require('../database-mongoose/reviews.service');
+let port = 4006;
 
 app.use(express.static('client/dist'));
 app.use(express.static('public'));
-//app.use(bodyParser.json());
-//app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.post('/Reviews', (req, res) => {
+  res.send('API post');
+
+});
+
+app.get('/Reviews', (req, res) => {
+  res.send('API get');
+
+});
+
+app.get('/Reviews/getReviews/:productId', (req, res) => {
+  let prodId = req.params; //{id: "5"}
+  db.getReviews(parseInt(prodId.productId))
+    .then(results => res.send(results))
+    .catch(err => console.log('err: ', err));
+});
+
+app.get('/Reviews/getReviewSummary/:productId', (req, res) => {
+  let prodId = req.params; //{id: "5"}
+  db.getReviewSummary(parseInt(prodId.productId))
+    .then(results => {
+      res.send(results)
+    }
+    )
+    .catch(err => console.log('err: ', err));
+});
+
+app.get('/Reviews/getReviewsByFeature/:productId', (req, res) => {
+  let prodId = req.params; //{id: "5"}
+
+  db.getReviewsByFeature(parseInt(prodId.productId))
+    .then(results => {
+      res.send(results);
+    })
+    .catch(err => console.log('err: ', err));
+});
+
+app.get('/Reviews/getReviewExcerpts/:productId', (req, res) => {
+  let prodId = req.params; //{id: "5"}
+  db.getReviewExcerpts(parseInt(prodId.productId))
+    .then(results => res.send(results))
+    .catch(err => console.log('err: ', err));
+});
```

Comment:
This looks good! One minor suggestion would be to set up the server so that it handles cases were an invalid id is requested or one that does not exist. I'd also add status codes along with your responses i.e. `res.status(...).send(...)`

Reply:
Good catch Alek. Will take care of it when I am working on unit tests - for next PR.